CMakeLists.txt

```diff
@@ -472,6 +472,7 @@ if(EXISTS ${CMAKE_BINARY_DIR}/zephyr_modules.txt)
 
     string(TOUPPER ${module_name} MODULE_NAME_UPPER)
     set(ZEPHYR_${MODULE_NAME_UPPER}_MODULE_DIR ${module_path})
+    set(ZEPHYR_${MODULE_NAME_UPPER}_MODULE_DIR ${module_path} PARENT_SCOPE)
```

Comment:
Won't the setting in the parent scope be available in this scope too? Are you just being redundant in case it's been set some other way for some reason?

Reply:
nope, it's not, so both are needed.

> If the PARENT_SCOPE option is given the variable will be set in the scope above the current scope. Each new directory or function creates a new scope. This command will set the value of a variable into the parent directory or calling function (whichever is applicable to the case at hand). The previous state of the variable’s value stays the same in the current scope (e.g., if it was undefined before, it is still undefined and if it had a value, it is still that value).

https://cmake.org/cmake/help/latest/command/set.html#set-normal-variable

Reply:
Ah, I see. Thanks for the explanation and pointer.